src/OpenTelemetry/CHANGELOG.md

```diff
@@ -13,6 +13,11 @@
   state for cumulative temporality.
   [#5230](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5230)
 
+* Fixed an issue causing `LogRecord`s to be incorrectly reused when wrapping an
```

Comment:
Could you specify if there is a way to detect if someone is running in to this issue?

Reply:
There isn't a super easy way to detect the issue. Records rented from the `ThreadStatic` pool have their attributes cleared when returned so most users will observe log records without attributes. But there could be a race too where you would get data from some other log. I added "which leads to missing or incorrect data during export" to the CHANGELOG entry how's that?

Reply:
This is more clear. Thanks!